### DISCUSSION TITLE
Error Creating a listing

### DISCUSSION THREAD

**Discussion by wainesiro** [ProjectOpenSea/opensea-js#1484] [Q&A]
When i try to create a listing using opensea sdk it works perfectly for some nft and for some nfts i keep getting this error "createListing failed error (attempt 1): Error: Server Error: An incorrect OpenSea fee has been provided. Expecting 250 basis points, but received 249." surely inside error because i didnt provide an opensea fee

**Comment by ryanio** [comment]
seems like a rounding error based on the listing price you are trying, try to bump the price up or down slightly. 

an opensea fee must be provided for the listing to be accepted by the opensea api so the sdk automatically formats this for you when creating an order
